src/main/java/core/basesyntax/dao/StorageDaoImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.dao;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class StorageDaoImpl implements StorageDao {
+    private final Map<String, Integer> fruits = core.basesyntax.db.Storage.fruits;
+
+    @Override
+    public Map<String, Integer> get() {
+        return Collections.unmodifiableMap(fruits);
+    }
```

Comment:
why do you need this class if I can access the storage from anywhere in the project? If you have any questions let’s  discuss them in the chat on the platform